### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Clorasun** [DUpdateSystem/UpgradeAll#263] [Q&A]
I don't know why I can't detect the latest version when I check for an updated application is it a problem with the application source?

**Comment by xz-dev** [comment]
> I don't know why I can't detect the latest version when I check for an updated application is it a problem with the application source?

Which app hub do you use?

**Reply by Clorasun** [reply]
https://www.helloimg.com/images/2022/09/14/ZIVo2A.png

**Reply by xz-dev** [reply]
> https://www.helloimg.com/images/2022/09/14/ZIVo2A.png

Google play uses Grayscale Release by default to release software, which is why you may not have detected the latest version.

**Reply by Clorasun** [reply]
Cool Market, Mobile Paradise both software sources, is not invalid, I can detect software updates within the application. In addition no response log how to extract my just now the application is not responsive, I feel that there are bugs caused by

**Reply by xz-dev** [reply]
So what software did you observe?

**Reply by Clorasun** [reply]
WeChat Reading，taobao，wps office，WeChat，Universal remote control，Alibaba Cloud Disk
